### ISSUE TITLE
Not Accessible to screen readers

### ISSUE THREAD

**Issue by carolyncole** [samvera/browse-everything#73]
Currently the screen reader will access the dialog, but does not get to any of the buttons or links in the dynamic content.

That means that the user can not authorize with oauth, and even after that happens they have no way of selecting or interacting with the file list.

I have noted in my updates with sufia that you can add a tab-index to items to make the screen reader find them, but have not gotten a chance to test it in this instance.
